app/assets/javascripts/discourse/components/composer-editor.js.es6

```diff
@@ -53,10 +53,15 @@ export default Ember.Component.extend({
   _xhr: null,
   shouldBuildScrollMap: true,
   scrollMap: null,
+  uploadPosition: null,
 
-  @computed
-  uploadPlaceholder() {
-    return `[${I18n.t("uploading")}]() `;
+  @computed("uploadPosition")
+  uploadPlaceholder(uploadPosition) {
+    if (uploadPosition) {
+      return `[${I18n.t("uploading")} (${uploadPosition})]() `;
```

Comment:
Maybe the `uploadPosition` should be in the translation? That way, other languages can put it where it's best for them.